### PR TITLE
Avoid keyword as var name.

### PR DESCRIPTION
**Pull request by k82cn**
fixes typo

### CODE REVIEW

**Comment by k82cn** [comment]
@k8s-bot gci gke e2e test this

File: pkg/registry/generic/registry/store.go [hunk 1]
```diff
@@ -402,8 +402,8 @@ func (e *Store) Update(ctx api.Context, name string, objInfo rest.UpdatedObjectI
 		if err := rest.BeforeUpdate(e.UpdateStrategy, ctx, obj, existing); err != nil {
 			return nil, nil, err
 		}
-		delete := e.shouldDelete(ctx, key, obj, existing)
-		if delete {
+		shouldDelete := e.shouldDelete(ctx, key, obj, existing)
```
Comment (smarterclayton) [inline pkg/registry/generic/registry/store.go]: We don't even need a variable here, just make this `if e.shouldDelete(...) {`

**Comment by k82cn** [comment]
@k8s-bot cvm gce e2e test this

**Comment by k82cn** [comment]
@smarterclayton , done.

**Pull request merged**